tests/cucumber/features/step_definitions/accounts_steps.coffee

```diff
@@ -0,0 +1,66 @@
+do ->
+  'use strict'
+
+  _ = require('underscore')
+
+  module.exports = ->
+
+    url = require('url')
+
+    @Given /^I am a new user$/, ->
+      @server.call('reset')
+
+    @When /^I navigate to "([^"]*)"$/, (relativePath, callback) ->
+      @client
+        .url(url.resolve(process.env.ROOT_URL, relativePath))
+        .call(callback)
```

Comment:
We'll have to be really careful to check in future features. The step will work no matter where it is, so it's easy to not notice you should move it, and the step files can get massive so it's hard to find anything.

Would it be crazy to put labels on the steps?
```
Given I am a new user /accounts
When I navigate to "xyz" /navigation
```
Of course the labels could get out of date too. How do people normally manage their cucumber steps?